md/release-notes.md

```diff
@@ -1,14 +1,62 @@
 # Release notes
 
-Bonita BPM 7.5 includes a set of user-based product enhancements, bug fixes and important technology updates.
+Bonita BPM 7.5 includes a set of new features related to the Living Application development and deployment and UI personalization in addition to important technology updates and bug fixes.
 
+<a id="living-application-development-and-deployment"/>
+
+## Living Application Development and Deployment
+
+### Integration of SSO SAML 2.0
+Bonita BPM works with Single-Sign-On (SSO) solutions using SAML V2 protocol `(Teamwork edition)`
+
+Capabilities: 
+* Being able to use standard authentication service 
+* Can connect to Bonita apps with my company credentials (per tenant configuration)
+* Common SAML 2.0 Identity providers : ADFS (MS Active Directory Federation Services), SalesForce, Open AM, Google Auth,...
+
+### New wizard to import a process in the studio & compare the artefacts 
+Provide a wizard when importing a .bos that display the duplicate artefacts (forms, connectors, ...)  and ask for actions to the users. Usual actions are proposed to user for every artefacts : keep existing or overwrite with the new import.
+
+Capabilities :
+* It eases collaboration between teams who works with .bos exchange
+* It eases the collaboration by file exchange and reduce drastically the risk of overwrite error during import of a .bos.
+
+### Industrialization of Living Application (in Studio)
+In Bonita BPM Studio, provide an editor to create the Living Application navigation without having to go on the Portal administration. This is the first step towards the full Application artefacts development into the Bonita BPM Studio in future versions `(Efficiency edition)`
+
+Capabilities : 
+* Create and edit LA navigation descriptor in Studio. It is the first step to industrialization of LA.
+* can construct application navigation from inside Studio based on existing artefacts.
```

Comment:
> existing artefacts

artefacts -> artifacts
explain also that existing means inside current workspace and not deployed on portal